docs/content/operator/iota-full-node/source.mdx

```diff
@@ -114,7 +114,23 @@ cp crates/iota-config/data/fullnode-template.yaml fullnode.yaml
 
     </Tabs>
 
-5. (Optional, skip this step to accept the default paths to resources). Edit the `fullnode.yaml` file to use custom paths.
+5. Set up archival fallback
+
+This allows nodes that fall behind to catch up by downloading archive data rather than relying on synchronization. For more details about archives, see [IOTA Archives](https://docs.iota.org/operator/archives)
```

Comment:
```suggestion
This allows nodes that fall behind to catch up by downloading archive data rather than relying on synchronization. For more details about archives, see [IOTA Archives](https://docs.iota.org/operator/archives).
```